src/pages/home/HomeRightbar.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useNavigate } from "react-router";
+import { Button } from "../../components";
+import dummy from "../../components/Header/dummy_profile_img.png";
+import { Loader } from "../../components/Loader/Loader";
+import { getAllUsers, followUser } from "../../reducers/userSlice";
+
+const HomeRightbar = () => {
+  const dispatch = useDispatch();
+  const { user, allUsers, fetchingAllUsersStatus } = useSelector(
+    (state) => state.user
+  );
+  const followSuggestionsArr = allUsers?.filter(
+    (ar) => !user?.following?.find((rm) => rm._id === ar._id)
+  );
+  const navigate = useNavigate();
+  // useEffect(() => {
```

Comment:
remove if not req

Reply:
this feature will be coming in future. That is why I have kept it.